src/main/drivers/timer_hal.c

```diff
@@ -86,7 +83,7 @@ static uint8_t lookupTimerIndex(const TIM_TypeDef *tim)
 #define _CASE(i) _CASE_(TIM##i##_BASE, TIMER_INDEX(i))
 
 // let gcc do the work, switch should be quite optimized
-    switch ((unsigned)tim >> _CASE_SHF) {
+    switch ((unsigned) tim >> _CASE_SHF) {
```

Comment:
It is unusual to use space between cast and variable

Reply:
Eclipse K&R formatting doesn't seems to work, I'll revert formatted code.

Reply:
Most formatting changes are OK and useful ...